libs/coin-modules/coin-evm/src/staking/detectOperationType.ts

```diff
@@ -40,8 +42,16 @@ const getStakingMethodSelectors = (
     if (!mapped || !fn) continue; // only map delegate/undelegate/redelegate
 
     try {
+      // Find the appropriate function in the ABI by the name
```

Comment:
unit test please 🙏 

also, is a try catch mandatory here ? parsing can go wrong ?